docs/userGuide/components/searchbar.md

```diff
@@ -34,10 +34,11 @@ To use the searchbar within a navbar, add the following markup to your file. The
 
 Name | Type | Default | Description
 ---- | ---- | ------- | ------
-data | `Array` || The local data source for suggestions. Expected to be a primitive array.
+data | `Array` || The local data source for suggestions. Expected to be a primitive array. To use the built-in search functionality, set this value to `"searchData"`.
 menu-align-right | `Boolean` | `false` | Whether the search bar's dropdown list will be right-aligned.
 on-hit | `Function` || A callback function when you click or hit return on an item.
 placeholder | `String` | `''` | The placeholder text shown when no keywords are entered in the search bar.
 
+%%{{ icon_warning }} Warning: If `data` is set to `"searchData"`, then `searchEnabled` **must be set to `true` in `site.json`**.%%
```

Comment:
Should we use `<box type="warning">` to be consistent with the other warnings in the documentation?
We can also link this to `siteComponents.html#enable-search`.